src/Components/Submitting.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef } from 'react'
+import { Modal } from 'react-bootstrap'
+import { connect } from 'scrivito'
+
+export const Submitting = connect(function Submitting() {
```

Comment:
How about renaming this component to something like `LoaderModal` or `FullscreenLoader`? My gut feeling tells me, that this could be used in other then submitting situations as well.